docs/tutorial.md

```diff
@@ -66,11 +66,11 @@ You need to configure this value only if you have cards with missing or incomple
 
 ### Learning and re-learning steps
 
-When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. This means that steps longer than or **equal** to 1 day should not be used. `23h` is not recommended because, while it's technically less than one day, it's very unlikely that you will be able to finish this step on the same day as your first review. In general, any steps longer than 12-14 hours are not recommended, since most people will not be able to finish such steps within the same day.
+When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. This means that steps longer than or **equal** to 1 day should not be used. `23h` is not recommended because, while it's technically less than one day, it's very unlikely that you will be able to finish this step on the same day as your first review. In general, any steps longer than 12-14 hours are not recommended, since most people will not be able to finish such steps within the same day. A reasonable learning step can be 10m, 15m, 20m or 30m.
```

Comment:
```suggestion
When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. In general, any steps longer than 12-14 hours are not recommended because most people will not be able to finish such steps within the same day. A reasonable learning step can be 10m, 15m, 20m or 30m.
```

Reply:
I added that part because some user thought that 23h is a good learning step

Reply:
This section is quite repetitive. The suggested wording retains the essence while significantly decreasing the length of the text.

Reply:
We need to clarify what "can be completed on the same day" means

Reply:
I have updated the comment. Have a look again.